### PR TITLE
Fix: Update tests

### PR DESCRIPTION
**Pull request by shaunorpen**
## Description
Update all tests which register a new user to delete that user before they're run again. This prevents errors to do with users already being registered when tests are run multiple times in a row. Also updated config so that all tests run sequentially rather than in parallel as otherwise one test might delete another's users when it's still running.
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works .
- [x] New and existing unit tests pass locally with my changes
## Link to Trello card
https://trello.com/c/x9V0sFEi/106-fix-broken-endpoint-tests

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -33,11 +33,7 @@ mongoose
     useFindAndModify: false,
     useCreateIndex: true
   })
-  .then(() => {
-    console.log('MongoDB is connected')
-  })
-
```
Comment (aapetsi) [inline server.js]: why are you deleting this?

Comment (shaunorpen) [reply server.js]: It didn't seem necessary and was messing up the output of my tests. If you don't get an error you've connected successfully, right?

Comment (aapetsi) [reply server.js]: it helps us know we connected successfully. i suggest we keep it

Comment (shaunorpen) [reply server.js]: Ben seems to be suggesting we remove it. Are you happy to, or still want to discuss further?

Comment (aapetsi) [reply server.js]: i'm good now. we can take it out

**Review by DimejiAre — APPROVED** [review]

**Review by aapetsi — APPROVED** [review]

**Pull request merged**